update-tests.sh

```diff
@@ -5,6 +5,7 @@ set +o errexit
 
 for test in $(find tests -name '*.ts'); do
   node src/index.js $test > $test.out 2>&1
+  rc=${PIPESTATUS[0]}; if [[ $rc != 0 ]]; then exit $rc; fi
```

Comment:
But for some of the tests (e.g. `tests/underscore-any.ts`), failure is expected. Does this script error out when you run it locally?